htc-api/api/htc_api.py

```diff
@@ -226,8 +238,10 @@ def get_synth_counties_stats():
     """Counties in JSON, statistics only synthetic"""
     log.debug("entering get_synth_counties_stats() IP=%s" % get_ip());
     con = get_db_con()
-    sql = "SELECT s.ct_fips, s.ct_name, s.sq_mi, s.pop, CASE WHEN s.pop > 0 THEN s.pop_male / s.pop ELSE 0 END AS pct_male, CASE WHEN s.pop > 0 THEN s.pop_female / s.pop ELSE 0 END AS pct_female, s.pop_sm " \
-      "FROM synth_ma.synth_county_stats s"
+    sql = "SELECT s.ct_fips, s.ct_name, s.sq_mi, s.pop, CASE WHEN s.pop > 0 THEN s.pop_male / s.pop ELSE 0 END AS pct_male, CASE WHEN s.pop > 0 THEN s.pop_female / s.pop ELSE 0 END AS pct_female, s.pop_sm, " \
+        "f_diabetes.rate / 100 as chr_diabetes " \
```

Comment:
I will normalize the rate to a value between 0 and 1 when updating records in the database. Please remove the div/100 here and anywhere else you query for rate so we're consistent.